picard/__init__.py

```diff
@@ -24,12 +24,26 @@
 PICARD_APP_NAME = "Picard"
 PICARD_VERSION = (2, 0, 4, 'dev', 1)
 
-
 # optional build version
 # it should be in the form '<platform>_<YYMMDDHHMMSS>'
 # ie. win32_20140415091256
 PICARD_BUILD_VERSION_STR = ""
 
+PROGRAM_UPDATE_LEVELS = {
```

Comment:
`picard.const` would be a better place imho.

Reply:
I was actually going to ask you about that.  I'll move it.